README.md

```diff
@@ -12,8 +12,6 @@ You can run the website locally using Hugo, or you can run it in a container run
 
 To use this repository, you need the following installed locally:
 
-- [yarn](https://yarnpkg.com/)
-- [npm](https://www.npmjs.com/)
```

Comment:
While you are making changes to this file, is it possible to remove the Go requirement?
Also, is npm required for contributors building and testing the site locally (no container build)?
See, https://github.com/google/docsy#prerequisites

Reply:
@kbhawkey I think that they are trying to unify to the containerized way? Maybe I'm wrong

Reply:
@inductor, I agree that the container-build bundles all of the requirements into one package.
However, I think it is worthwhile to document how to build the site with Hugo on the command line (hugo serve).
I happen to use both ways. The readme gets updated repeatedly which causes more toil for localization teams.

Reply:
Well yeah, I am in fact one of the owners of Japanese localisation... lol

Reply:
Does local development work OK even without NPM? I can believe that it might.

Reply:
@sftim Without container you need npm for local

Reply:
OK, then I'd retain `- [npm](https://www.npmjs.com/)`